tests/functional/tnf/test_tnf_failover.py

```
@@ -0,0 +1,330 @@
+
+@tier4b
+@turquoise_squad
+class TestTNFFailover:
+    """
+    TNF failover scenarios with workloads running.
+
+    Tests node failover with DRBD floating monitor migration
+    and static monitor pod failure recovery.
+    """
+
+    @pytest.fixture(scope="function")
```

Ruff (0.16.2)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)